src/stores/ServicesStore.js

```diff
@@ -100,6 +102,41 @@ export default class ServicesStore extends Store {
     );
   }
 
+  initialize() {
+    super.initialize();
+
+    // Check services to become suspended
+    this._suspendTick();
+  }
+
+  teardown() {
+    super.teardown();
+
+    // Stop checking services for suspend
+    this._suspendTick.cancel();
+  }
+
+  /**
+   * Сheck for services to become suspended.
+   */
+  _suspendTick = debounce(() => {
+    this._suspendServices();
+    this._suspendTick();
+  }, 1 * 60 * 1000); // every 1 min
```

Comment:
The problem with this approach is that you only take snapshots of which services are active every minute. This way you can miss a lot of interaction with services which should actually update their "last used" or "staleness" timer.

You need to introduce a `staleSince` timestamp for each service that is set everytime a service gets deactivated. Then you can do the suspend tick logic to check if the `staleSince` timestamp is older than e.g 5 minutes.